sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpSonarWayProfileTest.java

```diff
@@ -44,6 +44,7 @@ public class CSharpSonarWayProfileTest {
 
   @Before
   public void reset() {
+    logTester.setLevel(LoggerLevel.DEBUG);
```

Comment:
Here `logTester` is set at debug level, whereas [here](https://github.com/SonarSource/sonar-dotnet/pull/7038/files#diff-7e70bea7b19abca7b3bd930e3313f7e2888dffc16d21c099595b2a9128981bc0R59), it is set at trace level. 
My understanding is that before they were at the same level, because the default was not `INFO`. Is it the case? If so, why?

What was the default level before? Change notes don't say that, only state the new behavior, and I could not find that piece of information by looking at [the current LogTester](https://github.com/SonarSource/sonar-plugin-api/blob/master/test-fixtures/src/main/java/org/sonar/api/testfixtures/log/LogTester.java).